sdk/storage/storage-blob/test/node/highlevel.node.spec.ts

```diff
@@ -77,7 +77,7 @@ describe("Highlevel", () => {
 
     fs.unlinkSync(downloadedFile);
     assert.ok(downloadedData.equals(uploadedData));
-  });
+  }).timeout(10 * 60 * 1000);
```

Comment:
How about we give a name to this constant time and use the name instead? It may save the maintenance effort in the future.

Reply:
I would like to merge this PR before tonight's test run to eliminate these flaky tests, but I support using a constant here in a second PR.